.github/workflows/CD.yaml

```diff
@@ -122,6 +122,8 @@ jobs:
           KEYCLOAK_LINKEDIN_SECRET: ${{ secrets.KEYCLOAK_LINKEDIN_SECRET }}
           MAILJET_API_SECRET: ${{ secrets.MAILJET_API_SECRET }}
           MAILJET_API_ID: ${{ secrets.MAILJET_API_ID }}
+          NOTIFICATION_RECEIVERS_DATA_OWNERSHIP_REQUEST: ${{ secrets.NOTIFICATION_RECEIVERS_DATA_OWNERSHIP_REQUEST }}
```

Comment:
What is the content of this secret?

Reply:
prob.: info@dataland.com

Reply:
Excellent question. I did not set these on our server environment yet. For now I would just put in the same values as for the ones for the bulk data request. But I need to find out which values these are. Probably Erik and as you said info@dataland.com.

Reply:
Sounds good to me. Maybe we can clarify that with Andreas and Siegfried.

Reply:
It is now basically set to this. On dev instance it is as I said in teams info@dataland.com and dummy@dataland.com

Reply:
ok